debug.log

```diff
@@ -0,0 +1,80 @@
+# Logfile created on 2016-09-05 13:26:07 +1000 by logger.rb/47272
```

Comment:
Hi @itinsley 
I added this inside the .gitignore file:
debug.log
But it still pushed the debug.log file to github.
Do I still have to de-select it when I want to add the file to be pushed to github?
I might just delete that debug.log file completely.